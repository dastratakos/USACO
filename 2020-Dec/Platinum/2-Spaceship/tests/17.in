60 59 60
101000101111111111101111011001101110110111011110110110000111
111011110110000000111011110100111110110011100011001111101101
110010010011110101001001111111111011001011111100110010001111
110111111100110001010110100001110011111011011100111111000010
111011100111101100010010011001100110110111101100100110111111
110111011111001000011001110101111101111111011011101101011110
001101000111010110000101111110111110110110110001010111110111
010111001111101100111011011101001001001011110111111100110111
010110111111000111100101001010111010110111111010100111101111
100100101100101011011011000001111101110110010000001011100111
110011011010011111101100010010000000111010111110011111111111
010110101110101101110100111110100011000011111010111111001110
101111101101011100110101111101101011010111010011101111111010
111111101101111011111111101111111100111110111000011111101101
111111110010110111010101010110111100110101110111011110100011
111111100110111110100111111001010011100110011100111011111100
111110100111110011111110000111100111111101101111100001001111
011100111111111011011110111111011111101111000010110011111101
111011111111010011111111111100010101101100100011100101101100
110011001011001101001110111101010111101111101110111000111111
111101110101110101111100101111110010011000101111101110000110
111011101111101011110110001101111011101001101110100011110100
001011111100001011111111111111101110110110111001110001110111
000101011010111011100011111110010001111101111101111101000001
101001100111111011101110010101101010010100111110110111010011
111011101011101110111110111101110011010111111111001111011111
110011110001010110111111011011011001111101011110000111111110
110101111111100011011001011111111110010111111111011011011111
111111111111111011101111111111010110001110100101111000101001
111101011111010111111011101110111111110101111100100110001101
000111100101111111101110111011001111101011001011111001011110
011101011101110011111111111101101000111101111101101101110011
100101001111111110101111111101101101100011110010001101111011
101101111100111111101100000011100111101111111110111101001000
111100111101011111111111111110011001101110001110011001111111
111110011100111111111111011011110010110001000111101110111111
011111111101111001011010111110111010001010001111111011100110
010101011101110011000111010000001111110100101101010101101111
101101101101111001011101010000111011111010101111101111011000
011111111110110111111111100100111100010100110101111010010001
010100111110101100110110010000111111110011010101111111101100
010111111110101101111100010011111111110111010110111011111111
101111110000100111010110101111111110010110111110001110010010
000011101010111011111011001101101111111111100110100110111011
011111111111111011010001011101011011110011101100111111111101
111011000111001001111111110111101011111110111011101111000101
011111110111000100101011111101110000100101111111111111001111
100111111100011010111100011001110111111110111101111011100101
111111011111111101111111110110111101010101101011111111110001
101111011110000000011110101111110001111111000011110111111101
110110011111100111110010101001110000001111001101101000111010
101111111010111111110110101100110101111101111110101011101011
011100110101011010000111101111111111110101001101101000110110
000001001111100101011011100101011011110100100001111100110011
101110111001101110100110101111101111111111001111111101101110
011101110111111011111110110111100110101110100011001100100110
110111001011110101111001100111111011101110101011011110111100
101001011001100001101110011110011111101101101110100101111111
011111101111111111110011100011111011101011101111010100010101
010110111101010111111011011110110111000010110111110110000100
5 29 3 6
3 25 5 1
2 59 3 38
2 54 16 18
58 57 3 17
3 33 48 22
5 44 3 57
2 16 26 39
2 34 5 22
4 27 55 58
3 15 5 42
4 54 5 54
45 13 2 6
4 23 50 20
2 39 2 25
5 46 2 50
2 19 3 38
5 50 2 43
3 31 4 20
15 58 3 35
2 27 5 24
5 52 1 36
2 59 4 26
4 34 3 10
1 54 5 35
4 13 2 18
26 54 2 42
1 6 3 43
5 10 3 21
1 16 4 36
5 15 4 24
5 5 4 27
1 12 5 48
5 30 25 12
5 4 2 59
3 15 10 29
2 55 2 27
17 30 1 50
31 23 2 20
59 3 4 35
5 51 5 48
3 52 44 13
2 22 4 48
17 27 1 59
22 1 2 59
2 55 1 8
4 16 19 42
5 46 2 1
3 6 4 4
2 26 4 6
1 11 3 12
38 5 4 24
5 41 15 51
2 1 1 40
5 58 1 27
42 49 5 20
2 39 5 37
7 60 5 39
29 39 21 28
4 42 4 42
