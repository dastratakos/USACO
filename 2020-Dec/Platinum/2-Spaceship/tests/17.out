834268515
791596185
376711358
800171923
118641927
984251929
755154254
577964499
638038790
247139796
613474913
238382422
582031585
660501496
701886668
50922969
227091787
698008834
800479977
827767788
634193204
859715438
672059174
965234571
133581582
191948462
591232075
550966910
179129989
387097773
43580263
187078944
886399385
967075066
238409832
191842118
807907337
16839967
701427186
522541028
641927027
110008633
260073507
484617560
502057044
765599873
341733272
184746387
711174307
877147365
98140542
639058268
898908963
647347516
178019474
395581236
355920932
155928434
717273536
553627847
